src/drivers/sqlite/sqlite-generate-schema.ts

```diff
@@ -116,12 +117,19 @@ function generateConstraintScript(con: DatabaseTableColumnConstraint) {
 }
 
 export default function generateSqlSchemaChange(
-  change: DatabaseTableSchemaChange
+  change: DatabaseTableSchemaChange & DatabaseTriggerSchemaChange,
+  type: "table" | "trigger" = "table"
 ): string[] {
   const isCreateScript = !change.name.old;
 
   const lines = [];
 
+  if (type === "trigger") {
+    return [
+      `CREATE TRIGGER ${escapeIdentity(change.name.new ?? "")} \n${change.when} ${change.operation} ON ${escapeIdentity(change.tableName)} \nFOR EACH ROW \nBEGIN \n\t${change.statement} \nEND`,
+    ];
+  }
+
```

Comment:
Move it to the TRIGGER function inside the Sqlite driver. It is short function anyway, not worth putting here